homeassistant/components/updater.py

```diff
@@ -58,12 +63,11 @@ def _load_uuid(hass, filename=UPDATER_UUID_FILE):
 
 
 def setup(hass, config):
-    """Setup the updater component."""
+    """Set up the updater component."""
     if 'dev' in CURRENT_VERSION:
         # This component only makes sense in release versions
-        _LOGGER.warning(('Updater component enabled in dev. '
-                         'You will not receive notifications of new '
-                         'versions but analytics will be submitted.'))
+        _LOGGER.warning("Updater component enabled in 'dev'. "
```

Comment:
Single strings are wrapped in single quotes and sentences (especially in logger messages) in double quotes to be in sync with other platform/components.